webapp/packages/plugin-connections/src/ConnectionForm/Options/ConnectionOptionsTabService.ts

```diff
@@ -304,6 +304,20 @@ export class ConnectionOptionsTabService extends Bootstrap {
 
       const properties = await this.getConnectionAuthModelProperties(tempConfig.authModelId, state.info);
 
+      properties.forEach(property => {
```

Comment:
Seems like we should not trim properties here, as its from backend
we should trim credentials that we get from ui form